### DISCUSSION TITLE
BERS=1

### DISCUSSION THREAD

**Discussion by andrerav** [gopro/labs#224] [Q&A] [answered]
Hi!

I just noticed that a new metadata QR command **BERS=1** has showed up in the docs, however the description doesn't say much about what this setting is and what effect it has. Could you share some more information on what this setting does?

**Comment by dnewman-gpsw** [comment, marked as answer]
Generally you don't want to use this.  This will bypass rolling shutter corrections, it is for people experimenting with post image stabilization, and want more to work for themselves. This is more for doing custom product shoots where the camera is attached to the thing so no stabilization is needed..

**Reply by andrerav** [reply]
Whoops, yep I definitely don't want to disable that :) Thank you for the clear explanation!
